api/server.js

```diff
@@ -125,6 +125,43 @@ server.put("/api/boards/:boardId/cards/:cardId", async (req, res, next) => {
   }
 });
 
+//api route pushes a comment to our array of stirngs in card
+server.post("/api/cards/:cardId/comments", async (req, res, next) => {
+  const cardId = Number(req.params.cardId); //get the object thats in params and make it a number. What is passed in param is our card id
+  const { message, author } = req.body;
+
+  if (!message)
+    return next({ status: "422", message: "Comment message is required" });
+
+  try {
+    const cardToAddComment = await prisma.card.findUnique({
+      where: { cardId },
+    }); //get card we want to update first doing this to check if it even exists
+
+    if (!cardToAddComment) {
+      return next({ status: "404", message: "card to update not found" });
+    }
+
+    const commentString = author
+      ? `${author}: ${message}`
+      : `Anonymous: ${message}`; //after we found card to update build comment string we are going to store. If there is an author append it otherwise just put anonymous
```

Comment:
You can simplify this with the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing):
```suggestion
    const commentString = `${author ?? 'Anonymous'}: ${message}`;
```

Reply:
Oh this is cool I didn't even know this operator existed!